cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -1,61 +1,146 @@
+   nav_depth: Level of menu nesting. 1 equals the root menu.
+
+   nav_items: An array of tuples containing menu link info.
 
    ========================================================================== #}
 
-{% macro _sub_nav(sub_nav_title, sub_nav_landing, sub_nav_items, sub_nav_media) %}
-<div class="o-mega-menu_subcontent" aria-expanded="false">
-    <div class="o-mega-menu_subcontent-wrapper">
+{% macro _nav_list( nav_depth, nav_items ) %}
+{%- for nav_group in nav_items %}
+<ul class="list
+           list__unstyled
+           {{ _classes( nav_depth, '-list' ) }}">
+    {% if nav_depth == 1 -%}
+    <li class="list_item
+               {{ _classes( nav_depth, '-item' ) }}">
+        {% import 'molecules/global-header-cta.html' as global_header_cta with context %}
+        {{ global_header_cta.render() }}
+    </li>
+    {%- endif %}
+    {%- for nav_caption,
+            nav_url,
+            nav_children in nav_group %}
+    <li class="list_item
+               {{ _classes( nav_depth, '-item' ) }}"
+        {{ 'data-js-hook=flyout-menu' if nav_children | count > 0 else '' }}>
+        {# TODO: Remove nav_depth < 2 check on has-children class when 3rd level transitions are in. #}
+        <a class="{{ 'u-link__disabled' if nav_url == '' else '' }}
+                  {{ _classes( nav_depth, '-link' ) }}
+                  {{ _classes( nav_depth, '-link__has-children' ) if nav_children | count > 0 and nav_depth < 2 else '' }}
+                  {{ _classes( nav_depth, '-link__current' ) if nav_url == request.path else '' }}"
+           {{ '' if nav_url == '' or nav_url == request.path else 'href=' + nav_url | e }}
+           {{ 'data-js-hook=flyout-menu_trigger' if nav_children | count > 0 else '' }}>
+            {{ nav_caption }}
+        </a>
+        {% if nav_children | count > 0 %}
+            {{ _nav( nav_depth | int + 1, nav_children, nav_caption ~ ' Overview', nav_url, vars.media_items ) }}
+        {% endif %}
+    </li>
+    {%- endfor %}
+</ul>
+{%- endfor %}
+{% endmacro %}
+
+{# ==========================================================================
+
+   _nav()
+
+   ==========================================================================
+
+   Description:
+
+   Creates menu content markup when given:
+
+   nav_depth:        Level of menu nesting. 1 equals the root menu.
+
+   nav_items:        An array of tuples containing menu link info.
+
+   nav_overview:     A string used for a sub nav overview title.
+
+   nav_overview_url: A string used for a sub nav overview title URL.
+
+   media_items:      A tuple of strings composed of HTML markup.
+
+   ========================================================================== #}
+
+{% macro _nav( nav_depth, nav_items, nav_overview, nav_overview_url, media_items ) %}
+<div class="{{ _classes( nav_depth ) }}"
+     aria-expanded="false"
+     data-js-hook="flyout-menu_content">
+
+    <div class="{{ _classes( nav_depth, '-wrapper' ) }}">
```

Comment:
Yeah, agreed, but since the two inside wrappers are two different widths I'm not sure how to combine them without dropping the width standard added by .wrapper
More info here:
https://github.com/cfpb/cfgov-refresh/pull/1399#discussion_r52537007